src/builder/QuarkBuilder.sol

```diff
@@ -377,19 +457,55 @@ contract QuarkBuilder {
         }
     }
 
+    function getWrapperCounterpartBalance(
```

Comment:
Part of me is having dilemma here. 
Cuz it may seems to be in TokenWrapper, but underlying it's actually doing something related to Account and made me want to put it into Account. 
But it's a bit too TokenWrapper-like when its in Account, but too Account-like when it's in TokenWrapper 🤔

Reply:
That's fine, we can put it here for now until we find a better home for it